cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -85,11 +114,24 @@ public static KafkaConnectCluster fromConfigMap(ConfigMap cm) {
         kafkaConnect.setHealthCheckInitialDelay(Integer.parseInt(data.getOrDefault(KEY_HEALTHCHECK_DELAY, String.valueOf(DEFAULT_HEALTHCHECK_DELAY))));
         kafkaConnect.setHealthCheckTimeout(Integer.parseInt(data.getOrDefault(KEY_HEALTHCHECK_TIMEOUT, String.valueOf(DEFAULT_HEALTHCHECK_TIMEOUT))));
 
+        JsonObject metricsConfig = Utils.getJson(data, KEY_METRICS_CONFIG);
+        kafkaConnect.setMetricsEnabled(metricsConfig != null);
+        if (kafkaConnect.isMetricsEnabled()) {
+            kafkaConnect.setMetricsConfig(metricsConfig);
+        }
+
+        Storage storage = new Storage(Storage.StorageType.EPHEMERAL);
+        kafkaConnect.setStorage(storage);
```

Comment:
why the `Storage` here and in the `fromAssembly`? It's not related to the metrics but represents how/where the data are saved and it's valid for Zookeeper and Kafka.